internal_packages/github-contact-card/README.md

```diff
@@ -5,7 +5,7 @@ Uses GitHub's public API to look up a GitHub user based on their email address,
 and then displays public repos and their stars.
 
 This example is a good starting point for plugins that want to display data from
-external sources in the sidebar.
+external sources in the sidebar. If you want to see some more advanced plugins, have a look through [all the internal plugins, here on Github](https://github.com/timtjtim/N1/tree/master/internal_packages).
```

Comment:
Did you mean to put `https://github.com/nylas/N1/tree/master/internal_packages` instead?